doc/source/data/getting-started.rst

```diff
@@ -1,229 +1,86 @@
 .. _datasets_getting_started:
 
-===============
-Getting Started
-===============
+Dataset Quick Start
+===================
+
+Ray Dataset is an abstraction over a list of Ray object references to *blocks*, with APIs for distributed data loading and processing. Each block holds a set of items and can be in format of either an `Arrow table <https://arrow.apache.org/docs/python/data.html#tables>`__
```

Comment:
```suggestion
A Ray :class:`Dataset <ray.data.Dataset>` is a distributed data collection, holding a list of Ray object references pointing to distributed data **blocks**, with APIs for distributed data loading and processing. Each block holds an ordered collection of items and can be in the format of either an `Arrow table <https://arrow.apache.org/docs/python/data.html#tables>`__
```

Reply:
Nice suggestion. I also like that we call out it's ordered.